spec/views/admin/finance/show.html.erb_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.describe 'admin/finance/show.html.erb' do
```

Comment:
maybe we can add a feature spec as well? curious around the balance your keeping between view specs vs feature specs

Reply:
> curious around the balance your keeping between view specs vs feature specs

Good question. Usually the features test that a process that needs multiple pages works and a view spec just ensures the right thing's on the page in a particular state. But I don't think there's a hard and fast rule, sometimes a feature works better as it also verifies the routing etc (or JS etc).